ArticleTemplates/liveblogTemplate.html

```diff
@@ -149,6 +149,8 @@ <h1 class="the-minute__headline selectable">__TITLE__</h1>
             <img src="__TEMPLATES_DIRECTORY__assets/img/loading.gif" alt="Loading…"/>
         </div>
 
+        __CONTRIBUTIONS_EPIC__
```

Comment:
Yeah I don't think we clearly documented what was expected, so that was a fair assumption! I just happen to have seen how it's done on the site (between blocks) and recall conversations about it being between blocks.